common/projectconfig/testdata/withMinVersion/ftl-project.toml

```diff
@@ -0,0 +1,16 @@
+module-dirs = ["a/b/c", "d"]
+external-dirs = ["e/f", "g/h"]
+ftl-min-version = "0.129.2"
+
+[modules.module.configuration]
+  githubAccessToken = "keychain://githubAccessToken"
+
+[modules.module.secrets]
+  encryptionKey = "inline://notASensitiveSecret"
+  companyApiKey = "op://devel/yj3jfj2vzsbiwqabprflnl27lm/companyApiKey"
+
+[commands]
+  startup = ["echo 'Executing global pre-build command'"]
+
+[executables]
+  ftl = "ftl"
```

Comment:
We can remove this. It was in a previous version of this `ftl-project.toml` but has been removed

Reply:
Done. Thanks for catching that :)